README.md

```diff
@@ -4,6 +4,47 @@ This repository contains several sample applications that demonstrate various Ok
 
 Each sample makes use of the [Okta React Native Library](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react-native).
 
+## Prerequisites
+
+Before running the samples, you will need the following:
+
+* An Okta Developer Account, you can sign up for one at <https://developer.okta.com/signup/>.
+* An Okta Application, configured for Native mode. This is done from the Okta Developer Console. After login, from the Admin dashboard, navigate to `Applications → Add Application`. Choose Native as the platform. Populate your new Native OpenID Connect application with values similar to:
+  * **Application Name** 
+    * Native OpenId Connect App (must be unique)
+  * **Login redirect URIs**
+    * `com.sampleapplication:/`
```

Comment:
I know this is consistent with our other mobile guides, but is there any better guidance for them on what to do in an actual production app?  Will this be understood by most developers, or will they make a lot of "com.sampleapplication" entries?

Reply:
Make sense. Here I just moved it up into the root level, how about wrap it with bracket, so devs may pay attention.